.github/workflows/push.yml

```
@@ -71,3 +71,33 @@ jobs:
         uses: codecov/test-results-action@v1
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
+
+  Linting and Formatting:
```

actionlint (1.7.4)

invalid job ID "Linting and Formatting". job ID must start with a letter or _ and contain only alphanumeric characters, -, or _

(id)